src/plugins/plot/configuration/XAxisModel.js

```diff
@@ -94,7 +94,7 @@ export default class XAxisModel extends Model {
    */
   defaultModel(options) {
     const bounds = options.openmct.time.bounds();
-    const timeSystem = options.openmct.time.timeSystem();
+    const timeSystem = options.openmct.time.getTimeSystem();
```

Comment:
this warning was popping up that we're violating our own deprecated code!

Reply:
yeah, we gotta practice what we preach! 😆